confignode/src/main/java/org/apache/iotdb/confignode/persistence/NodeInfo.java

```diff
@@ -197,30 +197,38 @@ public TSStatus registerDataNode(RegisterDataNodePlan registerDataNodePlan) {
   }
 
   /**
-   * Persist Infomation about remove dataNode
+   * Persist Information about remove dataNode
    *
    * @param req RemoveDataNodeReq
    * @return TSStatus
    */
   public TSStatus removeDataNode(RemoveDataNodePlan req) {
-    TSStatus result = new TSStatus(TSStatusCode.SUCCESS_STATUS.getStatusCode());
-    removeNodeInfo.removeDataNode(req);
```

Comment:
the inner class RemoveNodeInfo is useless, the state of RemoveDataNodePlan is persient by Porcudere

Reply:
fixed